docs/deployment/metrics.md

```diff
@@ -13,24 +13,25 @@ By default NFD Master and Worker expose metrics on port 8081.
-| `nfd_master_build_info`                           | Gauge     | Version from which nfd-master was built                 |
-| `nfd_worker_build_info`                           | Gauge     | Version from which nfd-worker was built                 |
-| `nfd_gc_build_info`                               | Gauge     | Version from which nfd-gc was built                     |
-| `nfd_topology_updater_build_info`                 | Gauge     | Version from which nfd-topology-updater was built        |
-| `nfd_node_update_requests_total`                  | Counter   | Number of node update requests received by the master over gRPC |
-| `nfd_node_updates_total`                          | Counter   | Number of nodes updated                                 |
-| `nfd_node_update_failures_total`                  | Counter   | Number of nodes update failures                          |
-| `nfd_node_labels_rejected_total`                  | Counter   | Number of nodes labels rejected by nfd-master            |
-| `nfd_node_extendedresources_rejected_total`       | Counter   | Number of nodes extended resources rejected by nfd-master |
-| `nfd_node_taints_rejected_total`                  | Counter   | Number of nodes taints rejected by nfd-master            |
-| `nfd_nodefeaturerule_processing_duration_seconds` | Histogram | Time taken to process NodeFeatureRule objects            |
-| `nfd_nodefeaturerule_processing_errors_total`     | Counter   | Number or errors encountered while processing NodeFeatureRule objects |
-| `nfd_feature_discovery_duration_seconds`          | Histogram | Time taken to discover features on a node                |
-| `nfd_topology_updater_scan_errors_total`          | Counter   | Number of errors in scanning resource allocation of pods. |
-| `nfd_gc_objects_deleted_total`                    | Counter   | Number of NodeFeature and NodeResourceTopology objects garbage collected. |
-| `nfd_gc_object_delete_failures_total`             | Counter   | Number of errors in deleting NodeFeature and NodeResourceTopology objects. |
+| Metric                          | Type      | Description                                                                |
+| ------------------------------- | --------- |----------------------------------------------------------------------------|
+| `nfd_master_build_info`         | Gauge     | Version from which nfd-master was built                                    |
+| `nfd_worker_build_info`         | Gauge     | Version from which nfd-worker was built                                    |
+| `nfd_gc_build_info`             | Gauge     | Version from which nfd-gc was built                                        |
+| `nfd_topology_updater_build_info` | Gauge     | Version from which nfd-topology-updater was built                          |
+| `nfd_master_node_update_requests_total` | Counter   | Number of node update requests received by the master over gRPC            |
```

Comment:
small nit: could you keep the columns aligned

Reply:
done